bugzoo/mgr/coverage.py

```diff
@@ -91,6 +95,7 @@ def resolve_path(fn_rel: str) -> str:
                 if lines != []:
                     files_to_lines[path] = lines
 
+        logger.debug("Traversing all files finished. Seconds passed: {}".format(timer() - t_start))
```

Comment:
Can we just get the time taken to traverse the files, rather than the amount of time passed since the start of the function call?